monitoring/prometheus/rules.yml

```diff
@@ -34,3 +34,15 @@ groups:
     labels:
       severity: major
       group: 'instance'
+
+- name: Worker-Down
+  rules:
+  - alert: WorkerDown
+    expr: (count by (queue_name) (num_submissions_in_queue{is_remote=0}) - count by (queue_name) (num_processed_submissions{is_remote=0}))  >= 5
+    for: 120m
```

Comment:
Can we please make it 60m?

Reply:
some challenges have 60m evaluation time for submission so I kept this for 120m. Otherwise I'll have to set up alerts for each challenge separately and do it every time a new challenge comes in.